audio/sound/UI-sounds/desktop.ini

```diff
@@ -0,0 +1,2 @@
+[LocalizedFileNames]
```

Comment:
What's this?

Reply:
Cut-Paste made it automatically for some reason